app/views/providers/application_merits_task/has_other_opponents/show.html.erb

```diff
@@ -23,11 +23,11 @@
         <% @legal_aid_application.opponents.order(:created_at).each do |opponent| %>
           <tr class="govuk-table__row">
             <td class="govuk-table__cell" id="opponent<%= opponent.id %>"><%= opponent.full_name %></td>
-            <td class="govuk-table__cell">Individual<%#= opponent.type %></td>
+            <td class="govuk-table__cell"><%= opponent.type == "Individual" ? opponent.type : opponent.description %></td>
             <td class="govuk-table__cell">
               <%= link_to_accessible(
                     t("generic.change"),
-                    providers_legal_aid_application_opponent_individual_path(@legal_aid_application, opponent),
+                    opponent.type == "Individual" ? providers_legal_aid_application_opponent_individual_path(@legal_aid_application, opponent) : providers_legal_aid_application_opponent_organisation_path(@legal_aid_application, opponent),
```

Comment:
https://github.com/ministryofjustice/laa-apply-for-legal-aid/blob/main/app/helpers/state_benefit_partial_url_helper.rb

Reply:
Thanks @colinbruce that seems like a nice pattern. I have added an OpponentUrlHelper, so that the view can simply call oponent_url with the opponent type if that covers what you suggest?